app/src/main/res/values/arrays.xml

```diff
@@ -1,20 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
-    <string-array name="themesEntries">
+    <string-array name="themesEntries"> //not in use at the moment
```

Comment:
XML comments `<!-- look like this -->`; you added an actual string value in the XML that violates the XML schema and just happens to not cause any side-effects. (Same for all the other occurrences of the "pattern").